OneCommunityGlobal/HighestGoodNetworkApp
# Jiayu --> Demi_move_test_cases

# Description
Moves tests into their correct place within the file structure to get a more streamlined and consistent codebase.

## Related PRS (if any):
None

## Main changes explained:
- Moved tests into their appropriate folders
- Renamed incorrect folder names

## How to test:
1. check into current branch
2. view file structure and verify that all tests are within their individual components "__tests__" folder.

## Screenshots or videos of changes:

## Note:
